Gemfile

```diff
@@ -10,11 +10,11 @@ group :production do
 end
 
 group :development do
-  gem 'capistrano', require: false
-  gem 'capistrano-pending', require: false
-  gem 'capistrano-bundler', require: false
-  gem 'capistrano-rails', require: false
-  gem 'capistrano-passenger', require: false
+  # deployment
```

Comment:
You can leave the `require: false`s.  You don't need these gems in your load path to run the app.